kong/resty/getssl.lua

```diff
@@ -0,0 +1,23 @@
+local ffi = require "ffi"
+local pushssl = require "openssl.ssl".pushffi -- will define SSL* in ffi
+local get_ssl_pointer = require "ngx.ssl".get_ssl_pointer
+
+if get_ssl_pointer == nil then
+  error("Openresty patch missing. See https://github.com/Kong/openresty-patches")
```

Comment:
typo: OpenResty

More importantly: we should _gracefully_ fail if the patch is not available, but not prevent the module from loading altogether, which would certainly prevent Kong from even starting; while in fact, this feature might not even be used.

Reply:
I'm not sure how this *should* fail gracefully: if the patch is not applied and you're using this module, then you've likely got a security issue.

We *could* create a 'dummy' function that always returns `nil, "OpenResty patch missing"`. But would it get checked correctly? I guess that will do for now....